planner/core/point_get_plan.go

```diff
@@ -399,48 +405,64 @@ func checkFastPlanPrivilege(ctx sessionctx.Context, fastPlan *PointGetPlan, chec
 	return nil
 }
 
-func buildSchemaFromFields(ctx sessionctx.Context, dbName model.CIStr, tbl *model.TableInfo, tblName model.CIStr, fields []*ast.SelectField) *expression.Schema {
-	if dbName.L == "" {
-		dbName = model.NewCIStr(ctx.GetSessionVars().CurrentDB)
-	}
+func buildSchemaFromFields(dbName model.CIStr, tbl *model.TableInfo, tblName model.CIStr, fields []*ast.SelectField) (*expression.Schema, []*expression.NamingForMySQLProtocol) {
```

Comment:
the function signature is a little to long, can we split it to several lines? for example:
```go
func buildSchemaFromFields(
	dbName model.CIStr,
	tbl *model.TableInfo,
	tblName model.CIStr,
	fields []*ast.SelectField,
) (
	*expression.Schema,
	[]*expression.NamingForMySQLProtocol,
) {
```